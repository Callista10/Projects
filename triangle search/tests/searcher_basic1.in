-26.20, 4.42
-13.00, 10.39
12.57, -43.45
-48.68, 33.75
-24.06, -26.57