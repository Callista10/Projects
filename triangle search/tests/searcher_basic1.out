read 5 points
-26.20, 4.42
-13.00, 10.39
-24.06, -26.57
This is a triangle
